prometheus_client/core.py

```diff
@@ -198,6 +198,11 @@ def add_metric(self, labels, value):
           labels: A list of label values
           value: The value of the metric.
         '''
+        try:
```

Comment:
Ah yes, I wish I could think up an more Pythonic way.
I found this Question in StackOverflow
https://stackoverflow.com/questions/11301138/how-to-check-if-variable-is-string-with-python-2-and-3-compatibility
The best one needs **six**